Automattic/wp-calypso
# Checkout: Revert using Google Places API for smoother checkout

Reverts #34940

We did not see a compeling change in conversion, so let's revert for now.

#### Testing instructions

Try adding a domain to your site (signup or existing one, either way is the same). During checkout, you'll be prompted to enter your contact details.

Make sure that there are no regressions in the domain contact info form - try inputting some invalid data, etc. There should be no locations suggestions from Google, of course.

Test different countries:

 - no country - no address fields, can't submit
 - UK, EU and US countries (other countries default to US) have different layouts for the address fields, so check them all out as well

<img width="748" alt="Screenshot 2019-10-16 at 15 52 30" src="https://user-images.githubusercontent.com/3392497/66936294-fae43d80-f02c-11e9-9572-3428333e69a9.png">

<img width="745" alt="Screenshot 2019-10-16 at 15 52 22" src="https://user-images.githubusercontent.com/3392497/66936304-fd469780-f02c-11e9-8dd9-b7cdf5ac8c98.png">
